### PR TITLE
Update game to handle colorblind mode (black and white)

### PR DESCRIPTION
**Pull request by ALHennings**
Updates the visuals of the game to transition from normal to black and white designs in the Options menu. Maybe updated in future work. 

Closes issue #279 #304

### CODE REVIEW

File: client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/view/impl/HomeViewImpl.java [hunk 1]
```diff
@@ -104,6 +106,11 @@ private void playClicked() {
             showError(e.getMessage());
         }
     }
+    
+    private void colorblindClicked() {
+    	colorblind = !colorblind;
+    	reloadActiveGame();
```
Comment (tking2096) [inline client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/view/impl/HomeViewImpl.java]: You should check that there is an active game before trying to change it (NullPointerException).

**Review by tking2096 — DISMISSED** [review]

File: client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/view/impl/HomeViewImpl.java (hunk 1 above)
Comment (ALHennings) [reply client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/view/impl/HomeViewImpl.java]: Wow can't believe I missed that. Thanks.

Comment (ALHennings) [reply client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/view/impl/HomeViewImpl.java]: Nice catch. Fixed.

**Review by camja014 — APPROVED** [review]

**Review by tking2096 — APPROVED** [review]
Tested...👍

**Pull request merged**